apps/music/js/ui/views/player_view.js

```diff
@@ -300,6 +300,7 @@ var PlayerView = {
     } else {
       this.shuffleButton.classList.remove('shuffle-on');
     }
+    this.shuffleButton.setAttribute('aria-pressed', this.shuffleOption);
```

Comment:
@eeejay I don't know the `aria-pressed` attribute well, but do yo think we can merge it with the `shuffle-on` class in some way? so that we don't have to apply them speartely.